src/coreos-assembler

```diff
@@ -42,7 +42,7 @@ cmd=${1:-}
 # commands we'd expect to use in the local dev path
 build_commands="init fetch build run prune clean list"
 # commands more likely to be used in a prod pipeline only
-advanced_build_commands="buildprep buildupload oscontainer upload-oscontainer"
+advanced_build_commands="buildprep buildupload oscontainer upload-oscontainer release-ocp"
```

Comment:
Bikeshed: WDYT about `oc-adm-release`? That way it's clear that it's a wrapper around `oc adm release`. The use of `release` as a verb could be scary/misleading.